doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -930,6 +957,10 @@ Response code:
     + q: temperature>40 (optional, string) - A query expression, composed of a list of statements
       separated by `;`, i.e., q=statement;statements;statement.
       See [Simple Query Language specification](#simple_query_language). 
+
+    + mq: temperature.accuracy<0.9 (optional, string) - A query expression for attribute metadata,
+      composed of a list of statements separated by `;`, i.e., mq=statement;statements;statement.
```

Comment:
The second *statement* ends in an 's'. Probably not on purpose.

However, why not use;

    i.e., mq=statement1;statement2;statement3 
    
?

Also for 'q' (that might go to errata)